_posts/2019-03-03-march-agm.md

```diff
@@ -43,7 +43,7 @@ The positions available are as follows:
 
 #### What you need to do:
 
-- Send vicepresident@comp-soc.com your manifesto (max 200 words) **by Tuesday 4pm 19th March 2019**
+- Send vicepresident@comp-soc.com your manifesto (PDF of max 200 words) **by Tuesday 4pm 19th March 2019** (Please mention "AGM" or "Manifesto" in the email title to make it easier to find)
```

Comment:
Alternative solution is emailing `vicepresident+agm@comp-soc.com`, then all you need to do is a filter for `to:vicepresident+agm@comp-soc.com` (it's an email alias)